kolibri/plugins/learn/assets/src/views/TopicsPage.vue

```diff
@@ -57,45 +48,98 @@
           >
             {{ getTagline }}
           </KGridItem>
-
+        </KGrid>
+        <div class="tabs">
+          <KButton
+            ref="tab_button"
+            text="Folders"
```

Comment:
Use the new string `Folders` that you would have created in the [`commonLearnStrings.js`](https://github.com/learningequality/kolibri/pull/8424/files#r707883099) 😉